docs/types/value-types.rst

```diff
@@ -39,6 +39,11 @@ Operators:
 * Shift operators: ``<<`` (left shift), ``>>`` (right shift)
 * Arithmetic operators: ``+``, ``-``, unary ``-``, ``*``, ``/``, ``%`` (modulo), ``**`` (exponentiation)
 
+.. warning::
+
+  Integers in Solidity are restricted to a certain range. For example, with ``uint32``, this is ``0`` up to ``2**32 - 1``.
+  If the result of some operation on those numbers does not fit inside this range, it is truncated. These truncations can have
+  serious consequences, and code such as ``require((balanceOf[_to] + _value) >= balanceOf[_to])`` can help you check if values are what you expect.
```

Comment:
I think it is too dangerous to just mention this snippet here without any explanation. We should find a place to link to that properly explains how to deal with fixed-range integers and perform overflow checking. Do we already have something in the "security considerations"?

Reply:
You could link here :https://solidity.readthedocs.io/en/v0.5.2/security-considerations.html#two-s-complement-underflows-overflows and I just noticed that the link on that page is somehow ill-formatted.

Reply:
That's a good link, and I'll fix the rendering there too as it's related to this PR. But I wonder if I should move the example code and explanation here to the security doc and then just link from the warning to there?